### DISCUSSION TITLE
Improve PDF parser

### DISCUSSION THREAD

**Discussion by anusheel** [The-Black-Spatula-Project/black-spatula-project#6] [Ideas]
Currently we use pdfextract. This can be improved to use an LLM powered parse like LlamaParse or Firecrawl.dev etc.

**Comment by hypomariia** [comment]
https://lu.ma/b1yqug98 - recently found a great webinar

**Comment by fahminlb33** [comment]
The [olmOCR](https://olmocr.allenai.org/) model by Allen AI is pretty good for extracting data from PDF
